styles/Main.css

```diff
@@ -249,6 +250,9 @@ transition: transform 200ms ease;
   scale: 1.1;
 }
 @media screen and (max-width: 800px){
+  div.favourites label {
+    order: 0;
+  }
```

Comment:
This ensures that the favourites tab labels now appear immediately above their related content instead of in a group before all the tab content